brillig_vm/src/black_box.rs

```diff
@@ -219,6 +232,74 @@ fn verify_secp256k1_ecdsa_signature(
     public_key_y_bytes: &[u8; 32],
     signature: &[u8; 64],
 ) -> bool {
+    use k256::elliptic_curve::sec1::FromEncodedPoint;
+    use k256::elliptic_curve::PrimeField;
+
+    use k256::{ecdsa::Signature, Scalar};
+    use k256::{
+        elliptic_curve::{
+            sec1::{Coordinates, ToEncodedPoint},
+            IsHigh,
+        },
+        AffinePoint, EncodedPoint, ProjectivePoint, PublicKey,
+    };
+    // Convert the inputs into k256 data structures
+
+    let signature = Signature::try_from(signature.as_slice()).unwrap();
```

Comment:
We can just make an issue to remove code duplication for `verify_secp256k1_ecdsa_signature` and `verify_secp256r1_ecdsa_signature`

Reply:
Its commented at the function head